### PR TITLE
Add v1.5.0 of mattermost-plugin-memes to the Marketplace

### PR DESCRIPTION
**Pull request by hanzei**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-memes/issues/60

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -11517,6 +11517,56 @@
     },
     "updated_at": "2021-01-15T11:07:27.026366282Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-memes",
+    "icon_data": "",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-memes-v1.5.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-memes/releases/tag/v1.4.0",
```
Comment (catalintomai) [inline plugins.json:11524]: Is is `1.4.0` or `1.5.0` here and below?

Comment (hanzei) [reply plugins.json:11524]: Good catch. There is something fishy with the release. Looking into it: https://community.mattermost.com/core/pl/cch8c5mnkbff9gsue4w3oibfbo

Comment (hanzei) [reply plugins.json]: Fixed

**Review by catalintomai — APPROVED** [review]

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tested upgrade path
- Tested fresh install patch
- no changes to icon
- Briefly spot checked new memes
- Version number is correct at `v1.5.0`

LGTM!

**Pull request merged**